after/syntax/rst.vim

```diff
@@ -0,0 +1,26 @@
+if dracula#should_abort('rst')
+    finish
+endif
+
+hi! link rstComment                             DraculaComment
```

Comment:
Please use built-in highlight groups where able.

```suggestion
hi! link rstComment                             Comment
```

Reply:
No problem. Sounds much better. I'll do the correction.

Reply:
FWIW, I suspect @dsifford was referring to those that have obvious mappings (`Comment`, `Todo`), but not all of them do (why use `String` or `Keyword`, for example?).

But if it makes sense to you all, works for me.

Reply:
I am sorry for the misunderstanding. I did more work then needed. But now that is done, I kinda like `Keyword` and `String`. This makes it easier for user to override these colors later.

Reply:
Given that the things you linked to Tag seem to be link-related, I would say that makes at least semantic sense to me.

Reply:
Thanks for your insight. I dont mind hacks in my personal Vim config, but I dont want add burden on future maintenance. I am glad it's not too bad!